tasks/db_deploy/orca_sql.py

```diff
@@ -179,24 +186,24 @@ def app_user_sql(user_password: str) -> TextClause:
         DO
         $$
         BEGIN
-            IF NOT EXISTS (SELECT 1 FROM pg_user WHERE usename = 'orcauser' ) THEN
-                -- Create orcauser
-                CREATE ROLE orcauser
+            IF NOT EXISTS (SELECT 1 FROM pg_user WHERE usename = '{user_name}' ) THEN
+                -- Create {user_name}
+                CREATE ROLE {user_name}
                     LOGIN
                     INHERIT
                     ENCRYPTED PASSWORD '{user_password}'
                     IN ROLE orca_app;
 
                 -- Add comment
-                COMMENT ON ROLE orcauser
+                COMMENT ON ROLE {user_name}
                     IS 'ORCA application user.';
 
-                RAISE NOTICE 'USER CREATED orcauser. PLEASE UPDATE THE USERS PASSWORD!';
+                RAISE NOTICE 'USER CREATED {user_name}. PLEASE UPDATE THE USERS PASSWORD!';
```

Comment:
We can remove `PLEASE UPDATE THE USERS PASSWORD!` from this line. This is a leftover since we set the password in line 194.

Reply:
Removed